docs/website/references/cli/defradb_client_index_delete.md

```
@@ -1,27 +1,27 @@
-## defradb client index drop
+## defradb client index delete
 
-Drop a collection's secondary index
+Delete a collection's secondary index
 
 ### Synopsis
 
-Drop a collection's secondary index.
+Delete a collection's secondary index.
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)